hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/downloader/block/transformer/FileAppendTransformer.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2025 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.mirror.importer.downloader.block.transformer;
+
+import com.hedera.mirror.common.domain.transaction.TransactionType;
+import jakarta.inject.Named;
+
+@Named
+public class FileAppendTransformer extends AbstractBlockItemTransformer {
```

Comment:
nit: None of the transformers need to be public and can be final
```suggestion
final class FileAppendTransformer extends AbstractBlockItemTransformer {
```

Reply:
Fixed